kubernetes/watch-keeper/resource.yaml

```
@@ -34,6 +34,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: NODE_EXTRA_CA_CERTS
+              value: /home/node/envs/watch-keeper-secret/cacerts.pem
```

I'm fine with this except for the fact that there will be a warning when the file doesnt exist (which will probably be the majority of the time)

[docs ref](https://nodejs.org/api/cli.html#cli_node_extra_ca_certs_file):
```
A message will be emitted (once) with process.emitWarning() if the file is missing or malformed, but any errors are otherwise ignored.
```

i think we can achieve the same result with an optional env.
```suggestion
            - name: NODE_EXTRA_CA_CERTS
              valueFrom:
                configMapKeyRef:
                  name: watch-keeper-config
                  key: NODE_EXTRA_CA_CERTS
                  optional: true
```

which means someone who wants to add extra certs will have to update both the `watch-keeper-config` with the ca certs path, as well as the `watch-keeper-secret` with the actual pem file. 

Please make the change and test it. thanks.